### PR TITLE
Made badge component

### PR DESCRIPTION
**Pull request by poojavishnoi**
Please ignore doc files since I just pulled those to host the website containing only badge component on the doc page

### CODE REVIEW

File: components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+.avatar_badge_container {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  border: 1px solid lightgray;
+  border-radius: 10px;
+  padding: 1rem;
+  margin: 2rem 0rem;
```
Comment (AtulHande03a) [inline components/Badge/badge.css:14]: margin: 2rem 0;

File: components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+.badge_on_icon img {
+  width: 3rem;
+  margin-left: 1rem;
+}
+
+.icon_badge {
+  position: absolute;
+  background-color: rgb(26, 205, 218);
+  color: white;
+  padding: 0.2rem 0.4rem;
```
Comment (AtulHande03a) [inline components/Badge/badge.css:76]: 0.2rem,0.4rem could be written in px. Also, there are varying padding and margins .. is it possible to make them consistent?? use of utility class for that could help, but if can't be done, its still fine

File: components/Badge/badge.css (hunk 1 above)
Comment (poojavishnoi) [reply components/Badge/badge.css:14]: Okay

File: components/Badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  margin: 1rem;
+  padding-bottom: 2rem;
+  border-bottom: 1px solid lightgray;
+}
+
+.avatar_badge_container {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  border: 1px solid lightgray;
```
Comment (ravikumar0403) [inline components/Badge/badge.css:11]: Use variables for colors

Comment (poojavishnoi) [reply components/Badge/badge.css:11]: Okay

File: components/Badge/badge.css (hunk 2 above)
Comment (poojavishnoi) [reply components/Badge/badge.css:76]: Okay

**Pull request merged**